packages/transport/src/core/CoreErrors.ts

```diff
@@ -14,6 +14,20 @@ class Relationships {
         );
     }
 
+    public alreadyExists(status: RelationshipStatus) {
+        return new CoreError(
+            "error.transport.relationships.alreadyExists",
+            `No new Relationship to the peer can be created as a Relationship in status '${status}' currently exists.`
+        );
+    }
+
+    public notYetDecomposedByPeer() {
```

Comment:
I'd rename the error to `relationshipNotYetDecomposedByPeer` or `relationshipNotDecomposedByPeer` for clarity.

Reply:
This error will be thrown as `error.transport.relationships.notYetDecomposedByPeer`, as `notYetDecomposedByPeer` is just the name of the error's suffix. Do you still want to rename it, so that `error.transport.relationships.relationshipNotYetDecomposedByPeer` will be thrown?

Reply:
I'd prefer it that way. 👍